spec/acceptance/deprecation_spec.rb

```diff
@@ -1,19 +1,28 @@
 #! /usr/bin/env ruby -S rspec
 require 'spec_helper_acceptance'
-require 'shellwords'
 
 describe 'deprecation function' do
-  before :each do
-    FileUtils.rm_rf '/tmp/deprecation'
+
+  if fact('operatingsystem') == 'windows'
+    test_file = 'C:/deprecation'
+    else
+    test_file = "/tmp/deprecation"
+  end
+
+  add_file_manifest = "\"deprecation('key', 'message') file { '#{test_file}': ensure => present, content => 'test', }\""
```

Comment:
can you make this look like this instead:

```
add_file_manifest = <<-EOS
  deprecation('key', 'message')
  file { '#{test_file}': 
    ensure => present,
    content => 'test', 
  }
EOS
```

It's just easier to read and maintain.

Same with `remove_file_manifest`.